src/criteria-definitions.ts

```diff
@@ -0,0 +1,344 @@
+/**
+ * Centralized definitions of all FOLIO technical council acceptance criteria
+ * This file serves as the single source of truth for all criterion IDs and their organization
```

Comment:
> @maccabeelevine If the goal is for this file to be the single source of truth mapping identifiers to criteria, then (maybe ironically) I think we should reject https://github.com/folio-org/tech-council/pull/104 adding those identifiers to the criteria document. Otherwise it will be easy to edit/add them by accident in only one place, and get out of sync.

That is a fair point. Since there was pushback in that PR, I am opting to close it. My problem is that only a very light summary of the criteria is in this repo. I would not want to copy the words over and have that be out of sync.

Reply:
OK. Let see how the tool goes. I will close out #104 for now.